scalableminds/webknossos
# Add text to auth token view

This PR adds a short explanation text to the auth token view. It also adjusts the documentation on the auth token's position in the menu.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- go to the auth token view 
- there should be a short text explaining what an auth token is and when it should be revoked as well as a link to https://docs.webknossos.org/reference/rest_api#authentication

### Issues:
- solves #3691

------
